client/src/Components/Mentors.js

```diff
@@ -5,59 +5,118 @@ import {
 	Route,
 	Link,
 } from "react-router-dom";
-import Questions from "../mockData/Questions.json";
+// import Questions from "../mockData/Questions.json";
 
 import "../App.css";
 import "../grid.css";
 
-export default function Mentors() {
-	const [fetchedQuestions,setfetchedQuestions]=useState(Questions);
+export default function Mentors(props) {
+	// const [fetchedQuestions,setfetchedQuestions]=useState(props.questions);
 	const [newQuizzQuestions, setNewQuizzQuestions]=useState([]);
+	const [refreshQuestions, setRefreshQuestions]=useState(false);
+
 	const [newQuizz,setNewQuizz]=useState(	{
-		id: 1,
-		name: "javaScrypt week 1",
-		publiShingDate: "2020-08-04",
-		questions: [],
+
+		name: "",
+		publiShingDate: "",
+		questions_id: [],
```

Comment:
This should probably be `questionIds`?

Reply:
Agree. this name change comes from backend database naming. i need to ask them to change on database first